diff --git a/package/network/services/dnsmasq/files/dhcp.conf b/package/network/services/dnsmasq/files/dhcp.conf
index b5968d7..9abac4d 100644
--- a/package/network/services/dnsmasq/files/dhcp.conf
+++ b/package/network/services/dnsmasq/files/dhcp.conf
@@ -3,7 +3,7 @@ config dnsmasq
 	option boguspriv	1
 	option filterwin2k	0  # enable for dial on demand
 	option localise_queries	1
-	option rebind_protection 1  # disable if upstream must serve RFC1918 addresses
+	option rebind_protection 0  # disable if upstream must serve RFC1918 addresses
 	option rebind_localhost 1  # enable for RBL checking and similar services
 	#list rebind_domain example.lan  # whitelist RFC1918 responses for domains
 	option local	'/lan/'
